src/plugin-accounts/window/avatarlistframe.cpp

```diff
@@ -0,0 +1,618 @@
+// SPDX-FileCopyrightText: 2018 - 2023 UnionTech Software Technology Co., Ltd.
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "avatarlistframe.h"
+
+#include "avatarcropbox.h"
+
+#include <DSlider>
+
+#include <QColor>
+#include <QDateTime>
+#include <QDragEnterEvent>
+#include <QFileDialog>
+#include <QHBoxLayout>
+#include <QImage>
+#include <QLabel>
+#include <QMimeData>
+#include <QPainter>
+#include <QPoint>
+#include <QScreen>
+#include <QScrollArea>
+#include <QStandardPaths>
+#include <QTimer>
+#include <QVBoxLayout>
+#include <QWidget>
+
+// 系统用户头像存放路径
+#define PERSON_DIMENSIONAL_PATH "lib/AccountsService/icons/human/dimensional"
+#define PERSON_FLAT_PATH "lib/AccountsService/icons/human/flat"
+#define ANIMAL_DIMENSIONAL_PATH "lib/AccountsService/icons/animal/dimensional"
+#define ILLUSTRATION_DIMENSIONAL_PATH "lib/AccountsService/icons/illustration/dimensional"
+#define EMOJI_DIMENSIONAL_PATH "lib/AccountsService/icons/emoji/dimensional"
```

Comment:
```suggestion
#define EMOJI_DIMENSIONAL_PATH VarDirectory "/lib/AccountsService/icons/emoji/dimensional"
```
顺便，按照规范换成 static const QString 或 QStringLiteral 而不是定义宏更好（其他地方也一样）